concent_api/core/validation.py

```diff
@@ -106,6 +106,23 @@ def validate_list_of_identical_task_to_compute(list_of_task_to_compute: List[mes
     return True
 
 
+def validate_list_of_identical_report_computed_task(list_of_report_computed_task: List[message.tasks.ReportComputedTask]):
```

Comment:
I would add one more general comment to this.
For me `validate_...` functions should not return anything. In case validation passes, nothing happens. If it doesn't, function should raise proper Exception.
If you need boolean returning functions, these should be in form of a question, i.e.
`are_report_computed_task_messageges_identical` or sth similar.
Otherwise, it is very misleading.

Reply:
Done